### PR TITLE
Add tuya commands CMD_GET_NETWORK_STATUS  and CMD_TEST_WIFI

### PR DESCRIPTION
**Pull request by benjaminnestler**
## Description:
Add two tuya commands:
- Test Wi-Fi functionality (scanning)
- Get the current network status

See current Tuya Wifi Protocol Spec for more information (https://developer.tuya.com/en/docs/iot/tuya-cloud-universal-serial-port-access-protocol?id=K9hhi0xxtn9cb)

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ ] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by benjaminnestler** [comment]
The checks only fail, because of uploading timeouts of build artifacts. I couldn't restart the pipelines.

**Comment by barbudor** [comment]
Thanks
@ortegafernando was also waiting for CMD_GET_NETWORK_STATUS in this thread: https://github.com/arendst/Tasmota/discussions/17625

File: tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino [hunk 1]
```diff
@@ -1427,6 +1435,32 @@ void TuyaSetWifiStrength(void) {
...
+  // MCU request the module if whose SSID is 'tuya_mdev_test' and returns the result and the signal strength in percentage.
+  uint8_t payload_buffer[2] = {
+    0x00,   //'tuya_mdev_test' wifi not found
+    0x00    //signal strength = 0
+  };
+  
+  String test_wifi_ssid = "tuya_mdev_test";
+  
+  if (WiFi.SSID() == test_wifi_ssid){
+    int32_t rssi = WiFi.RSSI();
```
[the remark this answers is not in the document]
Comment (barbudor) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: In fact I don't get the point of this command
This looks like something specific for Tuya testing environment
Why would we need that into a Tasmota device ?

[the remark this answers is not in the document]
Comment (ortegafernando) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: HI, here is the command 0E about "test wifi":

https://developer.tuya.com/en/docs/iot/tuya-cloud-universal-serial-port-access-protocol?id=K9hhi0xxtn9cb#title-20-Test%20Wi-Fi%20functionality%20(scanning)

[the remark this answers is not in the document]
Comment (ortegafernando) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: > In fact I don't get the point of this command This looks like something specific for Tuya testing environment Why would we need that into a Tasmota device ?

Yes, it is strange, seems like a test enviroment.

[the remark this answers is not in the document]
Comment (benjaminnestler) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: Yes @barbudor and @ortegafernando the documentation describes these command. My device (the MCU) uses this command. I don't know why, but I thought "okay lets implement and repsonse to this command". I could image that some devices use this tuya test wifi ssid to get themselfes into a test mode.

[the remark this answers is not in the document]
Comment (barbudor) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: No a very big deal but in order to safe some code flash bytes if would just drop the test on the SSID name and always return the default 0x00 0x00 message as no one is expected to have such test SSID at home.

[the remark this answers is not in the document]
Comment (arendst) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: Totally agree.

[the remark this answers is not in the document]
Comment (benjaminnestler) [reply tasmota/tasmota_xdrv_driver/xdrv_16_tuyamcu_v1.ino:1448]: Okay ... that was my first though and implementation ... I will change it back.

**Review by barbudor — APPROVED** [review]

**Pull request merged**